kolibri/utils/server.py

```diff
@@ -322,3 +325,46 @@ def get_urls(listen_port=None):
         return STATUS_RUNNING, urls
     except NotRunning as e:
         return e.status_code, []
+
+
+def installation_type():  # noqa:C901
+    """
+    Tries to guess how the running kolibri server was installed
+
+    :returns: install_type is the type of detected installation
+    install_type can be any of these strings:
+        - installed_pex:  A PEX file
+        - installed_dpkg: A Debian package installed manually
+        - installed_apt: A Debian package installed using apt (usually from a ppa)
+        - installed_windows: Windows Installer
+        - installed_whl: Binary created after building the sources
+        - installed_dev: Development server
+        - installed_unknown: this function has not been able to guess it
+    """
+    install_type = 'installed_unknown'
+    if len(sys.argv) > 1:
```

Comment:
If it's run from uWSGI, I don't think we have any any executable at all. Could the logic build around this? I think you should consider the input that I gave about a `VERSION_PKG` file generated by the installers, that can alleviate a lot of the conditions and make it easier to mock+test code :)

Reply:
Hi, 
1. yes, for uWSGI it will say `unknown` but this PR also adds the info about the web server used, which will say uwsgi. 
2. About `VERSION_PKG`: we can add it when it exists, but it doesn't. In any case having it would replace 3 lines of code by 1 line of code, the 3 lines used to check if the package was installed manually or using apt. The lines to ensure the package is installed should not be removed. I don't think it's very important, but in any case, we can't use it until `VERSION_PKG` exists. I would even say that's not a warranty: nobody avoids you to download the package from the ppa repository and install it manually ;)

Reply:
I don't think moving it to utils.version is a good idea, as this is utils.server information. Anyway I will do it if more people think it's better. In any case tests don't depend on the place where the functions are. I just want to know if the functionality is approved to do tests for them. I don't want to work on tests for functionalities that might be removed or changed before being merged.

Reply:
yes, let's pin down the expected behavior and afterwards we can add tests